fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/mv/MVUtils.java

```diff
@@ -64,4 +82,31 @@ public static String getMVColumnName(Column mvColumn, String functionName, Strin
         }
         return "mv_" + mvColumn.getAggregationType().name().toLowerCase() + "_" + queryColumn;
     }
+
+    public static Map<String, Expr> parseColumnNameToDefineExpr(OriginStatement originStmt) throws IOException {
```

Comment:
why put this util function in optimizer rule? it seems that this api is used by catalog?